tests/sample_schemas/bad_app/bad_sample.json

```diff
@@ -14,7 +14,8 @@
                         {
                             "paramType": "query",
                             "name": "content",
-                            "required": true
+                            "required": true,
+                            "type": "string"
```

Comment:
type: string was added as jsonschema was flaking with `no type present` error instead of intended `nickname not present` error.